package.json

```diff
@@ -14,13 +13,13 @@
     "@nomiclabs/hardhat-web3": "^2.0.0",
     "@openzeppelin/contracts": "^4.4.2",
     "@openzeppelin/contracts-upgradeable": "^4.5.2",
+    "@terminal-fi/celo-devchain": "git+https://github.com/soloseng/celo-devchain#with-db-option",
```

Comment:
nit: very innovative way to move forward despite delays, and do you think it'll be cool to fork this under the celo-org github organisation?

Reply:
We could do that if we want to maintain our version of the `devchain` npm package. I guess we should evaluate that depending on response time of the original packages maintainers.